From 532800ebf3c5262506625fbb9d03ad9266d668aa Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Fri, 8 Nov 2013 13:57:46 -0800
Subject: [PATCH 393/974] MIPS: OCTEON: Set the extended bits of DIDTTO too.

The field was expanded and made disjoint.  Set both parts.  On old
chips this is reserved as zero, so it works there too.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/cavium-octeon/setup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/cavium-octeon/setup.c b/arch/mips/cavium-octeon/setup.c
index 335c79d..7c71f4f 100644
--- a/arch/mips/cavium-octeon/setup.c
+++ b/arch/mips/cavium-octeon/setup.c
@@ -465,6 +465,7 @@ void octeon_user_io_init(void)
 	 * between 1x and 2x this interval. For example, with
 	 * DIDTTO=3, expiration interval is between 16K and 32K. */
 	cvmmemctl.s.didtto = 0;
+	cvmmemctl.s.didtto2 = 0;
 	/* R/W If set, the (mem) CSR clock never turns off. */
 	cvmmemctl.s.csrckalwys = 0;
 	/* R/W If set, mclk never turns off. */
-- 
2.6.2

